app/views/news/index.html.erb

```diff
@@ -26,7 +26,19 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<h2><%=l(:label_news_plural)%></h2>
+<% html_title(l(:label_news_plural)) -%>
+<%= toolbar title: l(:label_news_plural) do %>
+  <% if User.current.allowed_to?(:manage_news, @project) %>
+    <li class="toolbar-item">
+      <%= link_to new_project_news_path(@project), title: l(:label_news_new), id: 'new_news_link', class: 'button -alt-highlight' do %>
```

Comment:
I see, so your classes are alright and the dark green is correct.

In that case lets keep the existing behaviour, even though it is discussable...